tests/mmptest/src/BindingProjectTests.cs

```diff
@@ -3,67 +3,132 @@
 using System.Linq;
 using System.Text;
 using NUnit.Framework;
+using Xamarin.Utils;
 
 namespace Xamarin.MMP.Tests
 {
 	public class BindingProjectTests
 	{
-		static string BindingName (bool full) => full ? "XM45Binding" : "MobileBinding";
```

Comment:
This will conflict badly with https://github.com/xamarin/xamarin-macios/pull/3715 and I'll need to rebase https://github.com/xamarin/xamarin-macios/pull/3715 after this lands.